### DISCUSSION TITLE
A "Stream mode" with the feature to hide the country flag and something other

### DISCUSSION THREAD

**Discussion by Kotoki1337** [ppy/osu#22089] [Ideas] [answered]
Some countries or regions censor content, and some country flags will cause osu! lazer to not be able to stream within their region, to avoid this the streamer must cover their leaderboard and something else. It might be a good idea to add an optional "stream mode" to hide them.

We already have one related issue open before in #12547 but it's for **identity** stuff.

I had drawn up a new set of flags for osucnlive to bypass the censorship ([HarukaKinen/osucnlive-country-flag](https://github.com/HarukaKinen/osucnlive-country-flag)), but that was clearly not suitable for the production environment.

**Comment by peppy** [comment, marked as answer]
Please use OBS to do this.

**Reply by Kotoki1337** [reply]
But lazer has flags almost everywhere, and if you use OBS to cover them up, then lazer becomes impossible to stream at all, because you need to cover up almost everything

So I think hiding the flag directly in the game is the best option?

**Reply by peppy** [reply]
Dunno. Edit your game resources or something. This isn't something we can support.
